app/models/article.rb

```diff
@@ -550,6 +556,28 @@ def validate_collection_permission
     errors.add(:collection_id, "must be one you have permission to post to")
   end
 
+  def validate_second_user
+    return unless second_user_id
+    return unless second_user_id == user_id
+
+    errors.add(:second_user_id, "must not be the same user as the author")
+  end
+
+  def validate_third_user
+    return unless third_user_id
+    return unless third_user_id == user_id
+
+    errors.add(:third_user_id, "must not be the same user as the author")
+  end
+
+  def validate_additional_authors_must_not_be_the_same
+    return unless second_user_id
+    return unless third_user_id
+    return if second_user_id != third_user_id
+
+    errors.add(:base, "second and third user cannot be the same")
+  end
```

Comment:
we don't have this three validations and now that we'll have a FK I decided to add them in. The second and third authors can technically be the same or be equal to the article's author. We don't want that